keyboards/flehrad/bigswitch/keymaps/vial/keymap.c

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2018 QMK Contributors
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#define KC_OSX_EJECT 0x66
+#define LOCK_OSX LSFT(LCTL(KC_OSX_EJECT))
+#define SLEEP_OSX LALT(LGUI(KC_OSX_EJECT))
```

Comment:
```suggestion
#define LOCK_OSX LSFT(LCTL(KC_KB_POWER))
#define SLEEP_OSX LALT(LGUI(KC_KB_POWER))
```
This keycode is already defined:
https://github.com/vial-kb/vial-qmk/blob/b4cfa21cb6bb2b6490268b323eaae8fbc2a19707/quantum/keycodes.h#L193